doc/releases/reef.rst

```diff
@@ -17,6 +17,9 @@ This is the first stable release of Ceph Reef.
    https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030129. We will build as
    soon as this bug is resolved in Debian stable.
 
+   If you use ``cephadm`` to install and you do not intend to use Ceph from the
```

Comment:
the bug in question has been fixed in bookworm. see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=929685

Reply:
and https://metadata.ftp-master.debian.org/changelogs//main/c/ca-certificates-java/ca-certificates-java_20230710~deb12u1_changelog